test/wallet/index.ts

```diff
@@ -180,27 +186,300 @@ describe('Wallet', function () {
   })
 
   describe('signTransaction', function () {
-    const publicKey =
-      '030E58CDD076E798C84755590AAF6237CA8FAE821070A59F648B517A30DC6F589D'
-    const privateKey =
-      '00141BA006D3363D2FB2785E8DF4E44D3A49908780CB4FB51F6D217C08C021429F'
-    const address = 'rhvh5SrgBL5V8oeV9EpDuVszeJSSCEkbPc'
+    let wallet
+    let wallet2
+    let wallet3
+    let wallet4
```

Comment:
Why do you have 4 wallets in this testing framework? And you're still using other wallets in some of the tests.

Reply:
The main reason there are 4 wallets is because the existing tests used different keys/secrets, and it's challenging to regenerate the expected result independent of our implementation, and I figured more wallets would only add to the testing coverage rather than hurting it.

Reply:
I think it makes these tests a little messy. 

It should be easy enough to generate expected results via local rippled/a different library. Since these tests are currently passing, you know that the code isn't broken atm, so you could also just run the test locally with a different wallet and use that result in the expected result (not optimal, but fine in this case).